content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -42,16 +42,17 @@ cluster for mesh expansion, run the following commands on a machine with cluster
 
     {{< text bash >}}
     $ cd install/kubernetes/helm/istio
-    $ helm upgrade --set global.meshExpansion=true istio-system .
+    $ helm upgrade --set global.meshExpansion.enabled=true istio-system .
```

Comment:
Please fix the `--set global.meshExpansion=true` at line 37 as well.

Reply:
Oh my bad... It's not there.

Reply:
Done.